docs/index.md

```diff
@@ -54,100 +60,35 @@ Download and install the latest Kubernetes plugin:
 steampipe plugin install kubernetes
 ```
 
-### Configuration
```

Comment:
## Get Started

### Install

### Configuration
Include simplest example of using kube config with the default kube config file location and the default context (or if they need to specify one, use an example)

Additional configuration examples are included below and for a full list of configuration arguments, please see the default [configuration file](https://github.com/turbot/steampipe-plugin-kubernetes/blob/main/config/kubernetes.spc).

### Run a Query

Show steampipe query, the query, results

## Configuring Kubernetes Cluster Credentials
...